docs/source/examples/setup_your_own_data.ipynb

```diff
@@ -0,0 +1,862 @@
+{
```

Comment:
<p>It would be good to add a %%time here to get a sense of how long something like this takes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>